package.json

```diff
@@ -68,6 +68,12 @@
           "default": false,
           "scope": "resource"
         },
+        "github.rejectUnauthorized": {
```

Comment:
I would like to name this `allowUnsafeSSL`  and default it to `false`.
With that name its much more clear what it does.

Reply:
yes, I will modify it later